src/auth/auth-api-request.ts

```diff
@@ -417,11 +422,11 @@ export class FirebaseAuthRequestHandler {
       return Promise.reject(
         new FirebaseAuthError(
           AuthClientErrorCode.INVALID_ARGUMENT,
-          'CustomUserClaims argument must be a non-null object.',
+          'CustomUserClaims argument must be a nullable object.',
```

Comment:
"must be an object or null" sounds better.

Reply:
Done.